pkg/nuclide-diagnostics-ui/lib/main.js

```diff
@@ -135,18 +135,19 @@ class Activation {
         .switchMap(updater => (
           updater == null ? Observable.of([]) : updater.allMessageUpdates
         )),
-      this._state.filterByActiveTextEditor,
-      // https://github.com/gajus/eslint-plugin-flowtype/pull/131
-      // eslint-disable-next-line flowtype/space-after-type-colon
-      (featureConfig.observeAsStream('nuclide-diagnostics-ui.showDiagnosticTraces'):
-          Observable<any>),
+      // eslint-disable-next-line max-len
```

Comment:
Looks like I can't change the PR, maybe since I don't have push access to your branch?

Anyway I'm not super picky how you format it. We have a rule on the team that you can't nitpick formatting at review time. If you care enough about anything you can write a lint rule for it. So, anything that passes lint is okay by me.

I'd suggest something like this:

```
((featureConfig.observeAsStream('nuclide-diagnostics-ui.showDiagnosticTraces'): any):
  Observable<boolean>),
```